app/_src/networking/transparent-proxy/introduction.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Transparent Proxy
+content_type: how-to
+---
+
+{% assign docs = "/docs/" | append: page.version %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign tproxy = site.data.tproxy %}
+
+{% capture tproxy-config-reference-info %}
+For a full list of transparent proxy settings with examples and configuration options, see the [Transparent Proxy Configuration Reference]({{ docs }}/reference/transparent-proxy-configuration/).
+{% endcapture %}
+
+A transparent proxy is a server that intercepts network traffic going to and from a service without requiring any changes to the application code. In {{ Kuma }}, it captures this traffic and routes it to the [data plane proxy]({{ docs }}/production/dp-config/dpp/#data-plane-proxy), allowing [Mesh policies]({{ docs }}/policies/introduction/#policies) to be applied.
```

Comment:
I think it would be good to add this `transparent proxy` definition to our concepts page https://kuma.io/docs/2.9.x/introduction/concepts/ and link from concepts to this page for further information

Reply:
addressed